content_schemas/dist/formats/email_alert_signup/frontend/schema.json

```diff
@@ -293,7 +293,6 @@
         "email_alert_type": {
           "type": "string",
           "enum": [
-            "topics",
```

Comment:
I strongly suspect this has nothing to do with specialist topics. Although I've not dug into it enough to find out where this schema is being used (will do so and let you know) ....if for example on [this](https://www.gov.uk/email-signup?link=/world/afghanistan) page you click **confirm** you can see that the params contain a topic attribute. Ie in the domain of email alerts, topic != specialist_topic.

Reply:
Please, let me know ther result of your investigation. This was done by analogy to the other email stuff as suggested by the following comment (which is not directly related though): https://github.com/alphagov/publishing-api/blob/9760686d780a9eec2539ffa3912fbb5bce9fb830/content_schemas/formats/shared/definitions/email_alert.jsonnet#L29. If you think it's something different from the other one, I will revert this change.

Reply:
Also, we have contract tests, but if they cannot be relied on, please, let me know.

Reply:
Ah yes I think you're right 👍